runtime/kusama/src/lib.rs

```diff
@@ -109,7 +109,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("kusama"),
 	impl_name: create_runtime_str!("parity-kusama"),
 	authoring_version: 2,
-	spec_version: 9180,
```

Comment:
Why are we bumping this? I thought only Polkadot will receive 9181, kusama already has 9180 and will stay with it.

Reply:
we should bump it because this is a breaking change to the kusama runtime, and thus should be incremented?

Reply:
but will kusama ever be upgrade to 9181? 

either way, please make sure that the `FixCouncilDeposit` migration will be executed on polkadot.

Reply:
> either way, please make sure that the FixCouncilDeposit migration will be executed on polkadot.

Just realized that there will be a 9180 release, so this whole discussion is moot and we're good.